ee/modules/021-cni-cilium/images/egress-gateway-agent/internal/controller/egressgateway_controller.go

```diff
@@ -33,6 +34,10 @@ const (
 	finalizerKey       = "egress-gateway.network.deckhouse.io"
 )
 
+var (
+	forbiddenPrefixes = []string{"cilium_", "lxc"}
```

Comment:
"forbidden" is authorization term. Here we filter interfaces, so "excludePrefixes" is more correct.